module/applications/actor/base-sheet.mjs

```diff
@@ -643,6 +643,13 @@ export default class ActorSheet5e extends ActorSheet {
 
       // Active Effect management
       html.find(".effect-control").click(ev => ActiveEffect5e.onManageActiveEffect(ev, this.actor));
+
+      for ( const override of Object.keys(foundry.utils.flattenObject(this.actor.overrides)) ) {
+        html.find(`input[name="${override}"],select[name="${override}"]`).each((i, el) => {
+          el.disabled = true;
+          el.dataset.tooltip = "DND5E.ActiveEffectOverrideWarning";
```

Comment:
For fields that have an existing tooltip, do we want to take that into account? Maybe prepending the tooltip with it?

Reply:
I tried a version where we didn't replace existing tooltips, and a version where we did (this version). I felt that replacing the tooltip worked best from a UX perspective. I didn't try a version where we concatenate the values at all, but it feels like that would be too long and also read strangely.

Reply:
Curious if this method covers the case where the input field isn't present, such as spell slot overrides, where the input field is created by clicking the 'edit' icon.

Reply:
> Curious if this method covers the case where the input field isn't present, such as spell slot overrides, where the input field is created by clicking the 'edit' icon.

I will add some handling for this too 😩. Any others you can think of?

Reply:
Anything in Special Traits and other sub-applications?

Reply:
Those are already handled with the current version.